src/Compilers/Core/Portable/DiagnosticAnalyzer/DefaultAnalyzerAssemblyLoader.Desktop.cs

```diff
@@ -34,11 +34,10 @@ protected override Assembly LoadFromPathImpl(string fullPath)
                 AppDomain.CurrentDomain.AssemblyResolve += CurrentDomain_AssemblyResolve;
             }
 
-            return LoadImpl(fullPath);
+            var pathToLoad = GetPathToLoad(fullPath);
+            return Assembly.LoadFrom(pathToLoad);
         }
 
-        protected virtual Assembly LoadImpl(string fullPath) => Assembly.LoadFrom(fullPath);
```

Comment:
Is this used from any other base code?

Reply:
I think I deleted all the overrides and usages of this, but accidentally left the original declaration around as a vestige.